pyro/contrib/gp/models/vgp.py

```diff
@@ -0,0 +1,108 @@
+from __future__ import absolute_import, division, print_function
+
+import torch
+from torch.autograd import Variable
+from torch.distributions import constraints, transform_to
+import torch.nn as nn
+
+import pyro
+import pyro.distributions as dist
+from pyro.distributions.util import matrix_triangular_solve_compat
+
+
+class VariationalGP(nn.Module):
+    """
+    Variational Gaussian Process module.
+
+    This model can be seen as a special version of SparseVariationalGP model
```

Comment:
since this is a special case why not have more code reuse?

Reply:
Thanks, I will make vgp as a subclass of svgp.